### PR TITLE
Encoding fixes for old content

### PR DESCRIPTION
**Pull request by rodfersou**

### CODE REVIEW

File: src/collective/behavior/amp/browser/__init__.py [hunk 1]
```diff
@@ -122,7 +122,10 @@ def metadata(self):
         metadata['author']['@type'] = 'Person'
         metadata['author']['name'] = self.author
 
-        metadata['description'] = self.context.Description()
+        description = self.context.Description()
```
Comment (hvelarde) [inline src/collective/behavior/amp/browser/__init__.py]: don't we need to use `safe_unicode` here also?

Comment (hvelarde) [inline src/collective/behavior/amp/browser/__init__.py]: one line instead of four: `metadata['description'] = self.context.Description() or ''`

File: src/collective/behavior/amp/utils.py [hunk 2]
```diff
@@ -59,6 +59,7 @@ def remove_invalid_tags(self, el):
             logger.debug('<{0}> tag was removed'.format(tag.tag))
 
     def __call__(self, code):
+        code = safe_unicode(code)
```
Comment (hvelarde) [inline src/collective/behavior/amp/utils.py]: best to use the `safe_unicode` on the method that calls the transformer; then we can add an assertion here to guarantee its the right value.

Comment (rodfersou) [reply src/collective/behavior/amp/utils.py]: I don't agree.. this way the utility is able to work with more than one encoding

Comment (hvelarde) [reply src/collective/behavior/amp/utils.py]: best to use the `raw_encoded` property [from RichTextValue](https://github.com/plone/plone.app.textfield/blob/master/plone/app/textfield/value.py#L59-L65).

**Comment by rodfersou** [comment]
@hvelarde I didn't understand.. here it shows merged, bug the pull request is still waiting with other number..

**Comment by hvelarde** [comment]
I merged by mistake; we need to do more tests before; the output has to be `unicode` and not `str`.
